src/main/java/org/breedinginsight/brapi/v2/dao/BrAPIGermplasmDAO.java

```diff
@@ -241,6 +259,16 @@ private Map<String,BrAPIGermplasm> processGermplasmForDisplay(List<BrAPIGermplas
         return programGermplasmMap;
     }
 
+    private boolean expand_programGermplasmByFullName(ProgramEntity program, Map<String, BrAPIGermplasm> programGermplasmByFullName) throws ApiException {
+        List<BrAPIGermplasm> allProgramGermplasm = getRawGermplasm(program.getId());
```

Comment:
`getRawGermplasm` hits the cache, calling it from the method `processGermplasmForDisplay` which is used to populate the cache may be causing the issues I'm seeing locally and in the TAF run.

Reply:
I thought the original intent of this card was to grab the accession numbers from the pedigree string [programKey-accessionNumber] and use that to create the newPedigreeString rather than requiring the parent germplasm objects @timparsons? Although I'm not sure how it would create the name and UUID pedigree strings in that case.